content/x-ray/x-ray.md

```diff
@@ -26,3 +26,17 @@ Click on the image to zoom
 {{% /notice %}}
 
 
+#### Challenge
```

Comment:
I added ?slow=true to the end of ELB URL as below and didn't notice any difference both in random generator and in X-Ray traces. Am I doing this as expected? 
If yes, is the latency too minimal for this test case? Should we increase this to 5 sec? 

http://a7b30403ef11b11e88b26029df2b2b66-2099615585.us-west-2.elb.amazonaws.com/?slow=true